### DISCUSSION TITLE
Project from Figma to Zepp Os how to do?

### DISCUSSION THREAD

**Discussion by TediPower** [zepp-health/zeppos-forum#199] [Q&A]
I made a project according to the guide [https://www.youtube.com/watch?v=wJucYvrNzW0](url)
What should I do now? to have my design on your watch?

**Comment by MyoungXUE** [comment]
Hi, it sounds like you referenced our video tutorial and created a watch app user interface using the component library, which is pretty cool.

Next, you need to refer to the official documentation of Zepp OS https://docs.zepp.com/docs/guides/quick-start/, use the API to implement the entire program function, and after debugging and testing, you can release the application to the application market.
